src/igvfd/tests/test_schema_construct_library_set.py

```diff
@@ -128,3 +128,15 @@ def test_dependencies_construct_library_set(award, lab, testapp, gene_myc_hs,
             'large_scale_gene_list': tabular_file['@id']
         })
     assert res.status_code == 201
+    res = testapp.post_json(
+        '/construct_library_set',
+        {
+            'lab': lab['@id'],
+            'award': award['@id'],
+            'file_set_type': 'expression vector library',
+            'scope': 'control',
+            'control_type': 'control transduction',
+            'selection_criteria': ['TF binding sites'],
+            'control_file_sets': [base_expression_construct_library_set['@id']]
+        })
+    assert res.status_code == 201
```

Comment:
Can we add a "negative" test to make sure that attempting to specify gene list results in an error?